pkg/controller/jobframework/reconciler.go

```diff
@@ -572,66 +569,80 @@ func (r *JobReconciler) recordAdmissionCheckUpdate(wl *kueue.Workload, job Gener
 	}
 }
 
-// IsAncestorJobManaged checks whether an ancestor job is managed by kueue.
-func (r *JobReconciler) IsAncestorJobManaged(ctx context.Context, jobObj client.Object) (bool, error) {
-	ancestor, err := r.getAncestorJobManagedByKueue(ctx, jobObj)
-	if err != nil {
-		return false, err
-	}
-	return ancestor != nil, nil
-}
-
 // getAncestorWorkload returns the Workload object of the Kueue-managed ancestor job.
-func (r *JobReconciler) getAncestorWorkload(ctx context.Context, jobObj client.Object) (*kueue.Workload, error) {
-	ancestor, err := r.getAncestorJobManagedByKueue(ctx, jobObj)
-	if err != nil || ancestor == nil {
-		return nil, err
+func (r *JobReconciler) getAncestorWorkload(ctx context.Context, ancestorJob client.Object) (*kueue.Workload, error) {
+	if QueueNameForObject(ancestorJob) == "" {
+		return nil, nil
 	}
 	wlList := kueue.WorkloadList{}
-	if err := r.client.List(ctx, &wlList, client.InNamespace(ancestor.GetNamespace()), client.MatchingFields{indexer.OwnerReferenceUID: string(ancestor.GetUID())}); client.IgnoreNotFound(err) != nil {
+	if err := r.client.List(ctx, &wlList, client.InNamespace(ancestorJob.GetNamespace()),
+		client.MatchingFields{indexer.OwnerReferenceUID: string(ancestorJob.GetUID())},
+	); client.IgnoreNotFound(err) != nil {
```

Comment:
Is this changed, or just re-formatted? If re-formatted, please revert.

Reply:
Reverted